app/scripts/lib/assertion.js

```diff
@@ -5,11 +5,12 @@
 define(function (require, exports, module) {
   'use strict';
 
+  var Duration = require('durationJs');
   var jwcrypto = require('vendor/jwcrypto');
   var P = require('lib/promise');
 
-  var CERT_DURATION_MS = 1000 * 60 * 60 * 6; // 6hrs
-  var ASSERTION_DURATION_MS = 1000 * 3600 * 24 * 365 * 25; // 25 years
+  var CERT_DURATION_MS =  new Duration('6h').milliseconds();
+  var ASSERTION_DURATION_MS = new Duration('52w').milliseconds() * 25; //25 years
```

Comment:
@larissagaulia see if you can find a way to validate that `Duration` for `ASSERTION_DURATION_MS` works correctly. We want to make sure assertion is correctly set to  expire in 25 years.
You can add an assert to existing tests in `app/tests/spec/lib/assertion.js`. 

also see below how it is used (` expiresAt: Date.now() + ASSERTION_DURATION_MS`)